aws-ami-creation/README.md

```diff
@@ -0,0 +1,9 @@
+## Folder Structure
+
+- `hcl2-files/`: This directory contains various files related to packer.
+- `scripts/`: This directory contains scripts used by packer provisner for installing additonal packages/softwares.
+
+
+## Usage
+
+[Add instructions or usage guidelines here if applicable]
```

Comment:
We should how to build with cli and through the CI so other team members can build the AMIs too.